src/user/user.repository.ts

```diff
@@ -48,4 +47,40 @@ export class UserRepository {
       where: { id },
     });
   }
+
+  async createPat(userId: number, token: string) {
+    const test = this.prisma.pAT.create({
+      data: {
+        token,
+        user: {
+          connect: {
+            id: userId,
+          },
+        },
+      },
+    });
+    return test;
```

Comment:
you should be able to immediately return here. Also I think `test` field was for test purposes right? 😉